### PR TITLE
Fema 15 lidar announcement

### PR DESCRIPTION
**Pull request by ThorTheDestroyer**

### CODE REVIEW

File: content/markdown/news/2015-10-15-lidar-data-release-middle-brazos-palo-pinto-fema-project.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+thumbalt: A thumbnail for Middle Brazos Palo Pinto Lidar
+mainimage: static/images/updates/2015/middle-brazos-palo-lidar/fema_2015_60cm_middle_brazos_palo_pinto_overview.jpg
+mainimagealt: 3-D Lidar rendering of Middle Brazos Palo Pinto
+abstract:
+    A FEMA initiative to capture high-quality Lidar data for parts of Palo Pinto, Stephens, Hood, Johnson, Erath, Eastland, and Somervell.  County, Texas
+---
+
+The Texas Natural Resources Information System (TNRIS) is pleased to announce the release of newly acquired FEMA high resolution Lidar data for 932 square miles of North Texas. Aerial collection took place during January 03, 2015 and February 09, 2015 during the leaf-off season. The Risk Assessment, Mapping, and Planning Partners performed data acquisition/processing  as well as the  quality assurance and quality control (QA/QC).
+
+The primary purpose of this project was to develop a consistent and accurate surface elevation dataset derived from high-accuracy Light Detection and Ranging (LiDAR) technology for the FEMA Middle Brazos-Palo Pinto, TX LiDAR Project Area.
```
Comment (wilsaj) [inline content/markdown/news/2015-10-15-lidar-data-release-middle-brazos-palo-pinto-fema-project.md]: Lidar*

**Comment by ThorTheDestroyer** [comment]
Sorry, I'm getting closer to remembering to write it as Lidar.

**Comment by wilsaj** [comment]
no PrOBLEM

**Comment by ThorTheDestroyer** [comment]
Chris please post this to the website along with the Fema 15 data set that was added to the  data catalog.

**Comment by jseppi** [comment]
assigned to @christoball

**Pull request merged**